packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.js

```diff
@@ -97,12 +123,12 @@ function TranscribedLines ({ lines, scale, task, theme }) {
               label={<TooltipLabel fill={fills.transcribed} label={counterpart('TranscribedLines.transcribed')} />}
             >
               <ConsensusLine
-                role='img'
+                role='button'
                 aria-describedby={id}
                 aria-label={line.consensusText}
                 focusColor={focusColor}
-                onClick={e => createMark(line)}
-                onKeyDown={e => (e.key === 'Enter' && createMark(line))}
+                onClick={event => createMark(event, line)}
+                onKeyDown={event => (event.key === 'Enter' && createMark(event, line))}
                 tabIndex={0}
```

Comment:
My bad, but @shaunanoordin pointed out that lines are always focusable and interactive, no matter which step of the workflow you are on. They could be marked up as disabled (`aria-disabled="true"`), outside of the transcription task, so that there's no expectation that they can be clicked during question steps, say.

Reply:
I've added `aria-disabled` state to the `ConsensusLine`, but I'm confirming whether or not a volunteer should be able to see the `ConsensusPopup` or not on other tasks. It might be useful, since there is a final question task in the next step asking if the letter is finished.

Reply:
I've confirmed with @snblickhan and @beckyrother that the consensus should be able to be referenced on other step tasks, but that volunteers should only be able to make a new mark and transcription on the step with the transcription task. I'll need to make a minor change to the completed lines.

Reply:
Ok this piece is now ready to go